patches/server/0969-add-player-showRespawnScreen-method.patch

```diff
@@ -0,0 +1,33 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: chmodsayshello <chmodsayshello@hotmail.com>
+Date: Mon, 20 Mar 2023 20:47:02 +0100
+Subject: [PATCH] add player#showRespawnScreen method
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index 1bada55af5d16437da4d16f9ded55f88a6121eb4..96105e77588fb9345d535c69091207a436aacbb9 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -38,35 +38,7 @@ import net.minecraft.nbt.CompoundTag;
+ import net.minecraft.network.FriendlyByteBuf;
+ import net.minecraft.network.chat.Component;
+ import net.minecraft.network.chat.PlayerChatMessage;
++import net.minecraft.network.protocol.game.ClientboundPlayerCombatKillPacket;
```

Comment:
you need add the // Paper
or not use the import and use new net.minecraft.network.protocol.game.ClientboundPlayerCombatKillPacket for avoid this.